opus_codec_registration.src.html

```diff
@@ -131,6 +131,7 @@
 <xmp>
 dictionary OpusEncoderConfig {
   OpusBitstreamFormat format = "opus";
+  [EnforceRange] unsigned long frameDuration;
```

Comment:
Should probably be "long long" and in microseconds like all other time fields in WebCodecs.

Reply:
we do find duration in audioData is in microseconds,  
as described in #371 , 
`frameSize = numberOfChannels*sampleRate * frameDuration / 1000`
most audio encoder use millseconds as frame duration(opus ptime for example),  we can change it to micoseconds, then ,the frameSize Calculation will be:
`frameSize = numberOfChannels*sampleRate * frameDuration / 1000/1000` 

Thanks.

Reply:
@dalecurtis Do you see the new commit according to your comments? This is my first PR that need some modifications, I'm afraid of missing an important action.

Reply:
We can see the new commits. This issue will be discussed at [TPAC](https://www.w3.org/2022/09/TPAC/) next week, so we might hold off on merging this PR until those discussions happen.

Reply:
I defer to Thomas for the final decision, I was just driving by.

Reply:
@aboba, @padenot: any thoughts on microseconds vs milliseconds?

IIUC, the valid `frameDurations` in milliseconds are {2.5, 5, 10, 20, 40, 60} according to [the spec](https://www.rfc-editor.org/rfc/rfc6716#section-2.1.4). Leaving it in microseconds might give the impression of finer grain support.

Reply:
Not sure microseconds are necessary.  ptimes when expressed in ms are rounded up.